code/main/edge/buildEdgeRequestURL.brs

```diff
@@ -13,12 +13,22 @@
 
 ' ***************************** MODULE: buildEdgeRequestURL *******************************
 
-function _adb_buildEdgeRequestURL(configId as string, requestId as string, edgeDomain = invalid as dynamic) as string
+function _adb_buildEdgeRequestURL(configId as string, requestId as string, path as string, edgeDomain = invalid as dynamic) as string
     scheme = "https://"
     host = "edge.adobedc.net"
-    path = "/ee/v1/interact"
+    overridablePath = "/ee/v1/interact"
     query = "?configId=" + configId
 
+    if not _adb_isEmptyOrInvalidString(path) then
+        if not path.startsWith("/")
+            path = "/" + path
```

Comment:
let's not add logic to correct the path. We should define the format we want the path in and expect path will be passed accordingly. Similar to what we do in Mobile case.

Reply:
There's no harm in adding this logic. In Brightscript, it takes work to access the API docs when calling it from other files. I prefer to keep it. Thoughts?

Reply:
We can simply add it in the function comments. I don't think we need this extra logic. Even if they developer doesnt find the docs, they can simply validate the output.

Reply:
Hmm, it's a weird thing to let the API client validate the output. I saw a lot of similar APIs supporting this feature. It should be better to allow the API client to use any of the following formats: 
- `/path/to/a/place/` 
- `/path/to/a/place` 
- `path/to/a/place/`